schoolopy/main.py

```diff
@@ -46,7 +46,12 @@ def _post(self, path, data):
         :return: JSON response.
         """
         try:
-            return requests.post(self._ROOT + path, data=data, headers={'Authorization': self._oauth_header()}).json()
+            return requests.post(self._ROOT + path, data=data, headers={
```

Comment:
I feel like maybe we should figure out a more general method of generating these headers, since we're using them everywhere. Also, can't we use `json` instead of `data`?